interactive_engine/executor/runtime/src/store/ffi.rs

```diff
@@ -1455,8 +1455,46 @@ impl GraphPartitionManager for VineyardPartitionManager {
         return Some((partition_id as u32, vertex_id));
     }
 
-    fn get_vertex_id_by_primary_keys(&self, _label_id: LabelId, _pks: &[Property]) -> Option<VertexId> {
-        unimplemented!()
+    fn get_vertex_id_by_primary_keys(&self, label_id: LabelId, pks: &[Property]) -> Option<VertexId> {
+        if pks.len() != 1 {
+            return None;
```

Comment:
Consider logging the case? These is unsupported not some error right?

Reply:
btw, I'm not sure if None means an error case here. A better and clear way may be to return `Result<Option<VertexId>, E>`?

Reply:
Can `ffi` code return `Result`? I'm not even sure if it is right to return `None`. `ffi` should be used by other codes such as JAva/C, they do not have `Option` or `Result`. BTW, the unsupported message better be `warn` rather than  `error`

Reply:
Okay.